LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Tracy Sambu
```

Comment:
This is usually attributed to the main project maintainer, or the creator, as you can find in other repositories(https://github.com/rails/rails/blob/main/MIT-LICENSE). The reason for this is, since licences are very important and legally binding in software, they must be owned by the main maintainers. That way, they can change or tweak it for the benefit of the project, and contributors will be at ease to contribute without fear of retribution. 


Could you please edit that to this?

```suggestion
Copyright (c) Kaka Ruto
```

The commit will still be attributed to you.

Thank you very much!